packages/design/src/responsiveBreakpoints.css

```diff
@@ -1,3 +1,10 @@
+@custom-media --small-screens-and-below (width < 490px);
+@custom-media --small-screens-and-up (width >=490px);
+@custom-media --medium-screens-and-up (width >=768px);
+@custom-media --large-screens-and-up (width >=1440px);
+@custom-media --extra-large-screens-and-up (width >=1680px);
+
+/* Deprecated */
```

Comment:
Should we create a backlog ticket to replace these deprecated breakpoints in Atlantis?

Reply:
oh, yup!

Reply:
Added a subtask since it is a pretty simple change in atlantis